injected/src/utils.js

```diff
@@ -487,6 +487,13 @@ export function isUnprotectedDomain(topLevelHostname, featureList) {
     if (!topLevelHostname) {
         return false;
     }
+
+    // special-case for localhost (eg: testing)
+    if (topLevelHostname === 'localhost') {
```

Comment:
Is this just a short circuit? Doesn't it work anyway? If not can we make it work rather than have an additional branch for this?

Reply:
actually not needed right now, I'll remove to further reduce the change